tools/rna_tools/rna_commender/.shed.yml

```diff
@@ -0,0 +1,9 @@
+name: rnacommender
+owner: rnateam
+description: INFO-RNA is a service for the design of RNA sequences that fold into a given pseudo-knot free RNA secondary structure.
```

Comment:
@gianlucacorrado can you please adopt this file and insert some useful information here? You can do this via the web-interface.